service/test/dinstaller/manager_test.rb

```diff
@@ -84,6 +85,11 @@
   end
 
   describe "#config_phase" do
+    before do
+      allow(subject).to receive(:testing_question)
+      allow(software).to receive(:testing_question)
```

Comment:
Well, now you get me. I am no longer sure if that testing question is testing leftover or not. But in general I do not like it :)